File: .sdk-repos/orchestration-cluster-api-go/examples/message_signal.go
// Message and signal operations: correlate/publish messages and broadcast signals.
package examples

import (
	"context"
	"fmt"

	camunda "github.com/camunda/orchestration-cluster-api-go"
	openapi "github.com/camunda/orchestration-cluster-api-go/client"
)

func correlateMessageExample(ctx context.Context, client *camunda.CamundaClient) error {
	// region CorrelateMessage
	req := openapi.NewMessageCorrelationRequest("order-confirmed")
	req.SetCorrelationKey("order-42")
	req.SetVariables(map[string]any{"confirmedBy": "payment-service"})

	result, err := client.CorrelateMessage(ctx, *req)
	if err != nil {
		return err
	}
	fmt.Printf("%v\n", result)
	// endregion CorrelateMessage
	return nil
}

func publishMessageExample(ctx context.Context, client *camunda.CamundaClient) error {
	// region PublishMessage
	req := openapi.NewMessagePublicationRequest("order-confirmed")
	req.SetCorrelationKey("order-42")
	req.SetVariables(map[string]any{"confirmedBy": "payment-service"})

	result, err := client.PublishMessage(ctx, *req)
	if err != nil {
		return err
	}
	fmt.Printf("%v\n", result)
	// endregion PublishMessage
	return nil
}

func broadcastSignalExample(ctx context.Context, client *camunda.CamundaClient) error {
	// region BroadcastSignal
	req := openapi.NewSignalBroadcastRequest("cancel-all-orders")
	req.SetVariables(map[string]any{"reason": "maintenance"})

	result, err := client.BroadcastSignal(ctx, *req)
	if err != nil {
		return err
	}
	fmt.Printf("%v\n", result)
	// endregion BroadcastSignal
	return nil
}

func searchMessageSubscriptionsExample(ctx context.Context, client *camunda.CamundaClient) error {
	// region SearchMessageSubscriptions
	result, err := client.SearchMessageSubscriptions(ctx, *openapi.NewMessageSubscriptionSearchQuery())
	if err != nil {
		return err
	}
	for _, s := range result.GetItems() {
		fmt.Printf("%v\n", s)
	}
	// endregion SearchMessageSubscriptions
	return nil
}

func searchCorrelatedMessageSubscriptionsExample(ctx context.Context, client *camunda.CamundaClient) error {
	// region SearchCorrelatedMessageSubscriptions
	result, err := client.SearchCorrelatedMessageSubscriptions(ctx,
		*openapi.NewCorrelatedMessageSubscriptionSearchQuery())
	if err != nil {
		return err
	}
	for _, s := range result.GetItems() {
		fmt.Printf("%v\n", s)
	}
	// endregion SearchCorrelatedMessageSubscriptions
	return nil
}

File: .sdk-repos/orchestration-cluster-api-php/examples/message_signal.php
<?php

/**
 * Compilable usage examples for message, signal, and message-subscription operations.
 */

declare(strict_types=1);

namespace Camunda\Orchestration\Examples;

use Camunda\Orchestration\CamundaClient;

// region SearchCorrelatedMessageSubscriptions
/**
 * Search correlated message subscriptions.
 */
function search_correlated_message_subscriptions(CamundaClient $client, ?\Camunda\Orchestration\Api\Model\CorrelatedMessageSubscriptionSearchQuery $correlatedMessageSubscriptionSearchQuery = null): void
{
    $client->searchCorrelatedMessageSubscriptions($correlatedMessageSubscriptionSearchQuery);
}
// endregion SearchCorrelatedMessageSubscriptions

// region SearchMessageSubscriptions
/**
 * Search message subscriptions.
 */
function search_message_subscriptions(CamundaClient $client, ?\Camunda\Orchestration\Api\Model\MessageSubscriptionSearchQuery $messageSubscriptionSearchQuery = null): void
{
    $client->searchMessageSubscriptions($messageSubscriptionSearchQuery);
}
// endregion SearchMessageSubscriptions

// region CorrelateMessage
/**
 * Correlate message.
 */
function correlate_message(CamundaClient $client, \Camunda\Orchestration\Api\Model\MessageCorrelationRequest $messageCorrelationRequest): void
{
    $client->correlateMessage($messageCorrelationRequest);
}
// endregion CorrelateMessage

// region PublishMessage
/**
 * Publish message.
 */
function publish_message(CamundaClient $client, \Camunda\Orchestration\Api\Model\MessagePublicationRequest $messagePublicationRequest): void
{
    $client->publishMessage($messagePublicationRequest);
}
// endregion PublishMessage

// region BroadcastSignal
/**
 * Broadcast signal.
 */
function broadcast_signal(CamundaClient $client, \Camunda\Orchestration\Api\Model\SignalBroadcastRequest $signalBroadcastRequest): void
{
    $client->broadcastSignal($signalBroadcastRequest);
}
// endregion BroadcastSignal
